old/lib/LedgerSMB/Form.pm

```diff
@@ -1805,7 +1813,7 @@ sub get_name {
 
     # SC: Check for valid table/view name.  Other values will cause SQL errors.
     if ($table !~ /^(vendor|customer|employee)$/i) {
-        $self->error('Invalid name source');
+        $self->error('Invalid name source "$table"');
```

Comment:
This will return the string

```
Invalid name source "$table"
```

because single-quoted strings don't get their variables replaced.